src/editor/Core/ConfigurationParser.re

```diff
@@ -162,6 +162,10 @@ let configurationParsers: list(configurationTuple) = [
     "editor.zenMode.hideTabs",
     (s, v) => {...s, zenModeHideTabs: parseBool(v)},
   ),
+  (
+    "editor.zenMode.singleFile",
```

Comment:
You'll want to add this configuration option to the `ConfigurationDefaults.re` file as well, since that string is currently used to generate the config file on peoples system when they open it for the first time.

I had added a test case to try and alert people when they forgot to update it, but it looks like since I'm comparing the parsed config, it always has every option in it as parsing adds back any missing values... Guess I need to update that test to use the non-parsed config string which will be a pain.

Sorry about that!

Reply:
I added it, thanks for pointing that out :)